.travis.yml

```diff
@@ -0,0 +1,12 @@
+sudo: false
+language: scala
+scala:
+  - 2.12.3
+jdk:
+  - oraclejdk8
+script: sbt clean coverage test
```

Comment:
This is only one third of a coverage configuration. Either implement the coverage tests fully, or pull it out so it doesn't waste compute/memory.

1. Run tests with coverage.
2. Generate the coverage [report](https://github.com/scoverage/sbt-scoverage#how-to-use).
3. Push the report to a service like [codecov.io](https://docs.codecov.io/docs/supported-languages), or the older [coveralls](https://coveralls.zendesk.com/hc/en-us/articles/201342879-Scala).